benchmark/Dockerfile

```diff
@@ -20,27 +28,19 @@ RUN apt-get update && apt-get install -y \
 # Make python3.11 the default python3
 RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.11 1
 
-# Install Go with architecture detection
-RUN ARCH=$(uname -m) && \
-    if [ "$ARCH" = "x86_64" ]; then \
-        GOARCH="amd64"; \
-    elif [ "$ARCH" = "aarch64" ]; then \
-        GOARCH="arm64"; \
-    else \
-        false; \
-    fi && \
-    curl -L "https://golang.org/dl/go1.21.5.linux-$GOARCH.tar.gz" -o go.tar.gz && \
-    tar -C /usr/local -xzf go.tar.gz && \
-    rm go.tar.gz
+# Install Go
+COPY --from=go-base /usr/local/go /usr/local/go
```

Comment:
Maybe change it to `COPY --from golang:1.21.5  /usr/local/go /usr/local/go ` and remove the changes at this line: https://github.com/PGCodeLLM/aider/pull/2/files#diff-5b448215f5a2b2c7bd6e5c23a7a52bfd24454fe3b3c4045fefbac8313195f201R1